src/ephemeris/get_tool_list_from_galaxy.py

```diff
@@ -103,7 +103,9 @@ def record_repo(tool_elem):
         if self.get_data_managers:
             for tool in self.installed_tool_list:
                 if tool.get("model_class") == 'DataManagerTool':
-                    repositories.append(get_repo_from_tool(tool))
+                    repo = get_repo_from_tool(tool)
+                    if repo:
```

Comment:
This is a good idea if a tool does not have a repo. Could you add a comment here with an example of why this is needed? For the people who read the code in the future and do not know all galaxy specifics by heart.